L1Trigger/L1THGCal/src/be_algorithms/HGCalClusteringImpl.cc

```diff
@@ -435,3 +441,41 @@ void HGCalClusteringImpl::removeUnconnectedTCinCluster( l1t::HGCalCluster & clus
     }
     
 }
+
+
+
+void HGCalClusteringImpl::calibratePt( l1t::HGCalCluster & cluster ){
+
+  double calibPt=0.;
```

Comment:
The indentation is 4 spaces everywhere else in this file. It would be better to be consistent here.